go/chat/boxer.go

```diff
@@ -741,16 +741,7 @@ func (b *Boxer) unboxV2orV3orV4(ctx context.Context, boxed chat1.MessageBoxed,
 	// intended for our device, and error out if it's missing or invalid. If it
 	// is valid, then we *don't* validate the message signing key. That is,
 	// even though signEncryptOpen will check a signature in the end, we no
-	// longer care what signing key it's using. That means we're compatible
```

Comment:
What is wrong with this comment?

Reply:
we never ended up optionally including pairwise macs but pushed reader support for v4 when we initially released exploding messages. Once we fix the caching issues we'll release with writing pairwise macs on v4 since existing clients with exploding support have mac reader support